var/spack/repos/builtin/packages/py-htseq/package.py

```diff
@@ -10,15 +10,29 @@ class PyHtseq(PythonPackage):
     """HTSeq is a Python package that provides infrastructure to process
     data from high-throughput sequencing assays."""
 
-    homepage = "https://htseq.readthedocs.io/en/release_0.9.1/overview.html"
-    url = "https://github.com/simon-anders/htseq/archive/release_0.9.1.tar.gz"
+    homepage = "https://htseq.readthedocs.io/en/master/index.html"
+    pypi = "HTSeq/HTSeq-2.0.3.tar.gz"
 
-    version("0.11.2", sha256="dfc707effa699d5ba9034e1bb9f13c0fb4e9bc60d31ede2444aa49c7e2fc71aa")
-    version("0.9.1", sha256="28b41d68aa233fce0d57699e649b69bb11957f8f1b9b7b82dfe3415849719534")
+    version("2.0.3", sha256="c7e7eb29bdc44e80d2d68e3599fa8a8f1d9d6475624dcf1b9644285a8a9c0fac")
+    version("0.11.2", sha256="65c4c13968506c7df92e97124df96fdd041c4476c12a548d67350ba8b436bcfc")
+    version("0.9.1", sha256="af5bba775e3fb45ed4cde64c691ebef36b0bf7a86efd35c884ad0734c27ad485")
 
+    variant("qa", default=True, description="Quality assessment")
+    variant("mtx", default=True, description="BigWig manipulation", when="@2:")
+    variant("mtx", default=True, description="mtx output files", when="@2:")
+    variant("h5ad", default=True, description="h5ad output files", when="@2:")
+    variant("loom", default=True, description="loom output files", when="@2:")
```

Comment:
I usually disable these by default to minimize the number of dependencies, but I'll leave the final decision up to you

Reply:
I did give this some thought but left them enabled. In reality, I'd say these are 'pseudo requirements' rather than true optionals - although the readthedocs does explain, the relevant `argparse` options don't detail anything themselves ...
https://github.com/htseq/htseq/blob/e05ed6ed2cc498370f6cbcfe297a68a53cd2e7b8/HTSeq/scripts/count.py#L381
https://github.com/htseq/htseq/blob/e05ed6ed2cc498370f6cbcfe297a68a53cd2e7b8/HTSeq/scripts/count.py#L389... and I wouldn't expect endusers to wade through the docs to try and work out why their scripts are failing when running with 'standard' options.